src/index.html

```diff
@@ -9,10 +9,75 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Make sure to link to the compiled CSS file instead.